crates/cheatnet/src/forking/state.rs

```diff
@@ -201,21 +199,14 @@ impl StateReader for ForkStateReader {
                         value: BigUint::from_bytes_be(&field_element.to_bytes_be()),
                     })
                     .collect();
-                let converted_entry_points: ContractEntryPoints = serde_json::from_str(
-                    &serde_json::to_string(&flattened_class.entry_points_by_type).unwrap(),
-                )
-                .unwrap();
-                let converted_abi: Contract = serde_json::from_str(&flattened_class.abi).unwrap();
 
-                let sierra_contract_class: ContractClass = ContractClass {
-                    sierra_program: converted_sierra_program,
-                    sierra_program_debug_info: None,
-                    contract_class_version: flattened_class.contract_class_version,
-                    entry_points_by_type: converted_entry_points,
-                    abi: Some(converted_abi),
-                };
-                let casm_contract_class: CasmContractClass =
-                    CasmContractClass::from_contract_class(sierra_contract_class, false).unwrap();
+                let sierra_contract_class = serde_json::json!({
+                    "sierra_program": converted_sierra_program,
+                    "contract_class_version": "",
+                    "entry_points_by_type": flattened_class.entry_points_by_type
+                });
```

Comment:
I think it would be more readable if we created the `ContractClass` struct and then deserialized it to json, now it looks a bit like magic. If field names change it will be hard to find the issue

Reply:
Yeah, but here's why it is handled like that.
We do not know which Sierra version it is so we would have to perform some weird conversion

Reply:
Okay, I trust you